docs/self-managed/reference-architecture/manual/aws-ec2.md

```diff
@@ -0,0 +1,348 @@
+
+7. **Access outputs**
+
+After the infrastructure is created, you can access the outputs defined in `outputs.tf`. For example, to get the OpenSearch endpoint:
+
+```sh
+terraform output aws_opensearch_domain
+```
+
+8. **(optional) Connect to remote machines via Bastion host**
+
+The EC2 instances are not public and have to be reached via a Bastion host.
+Alternatively one can utilize the [AWS VPN Client](https://docs.aws.amazon.com/vpn/latest/clientvpn-admin/what-is.html) to connect securely to a private VPC. This is not covered here as the setup requires a lot more manual user interaction.
+
+```
+export BASTION_HOST=$(terraform output -raw bastion_ip)
+# retrieves the first IP from the camunda_ips array
+export CAMUNDA_IP=$(tf output -json camunda_ips | jq -r '.[0]')
+
+ssh -J admin@${BASTION_HOST} admin@${CAMUNDA_IP}
+```
+
+### Deploy Camunda 8
+
+1. **Go to the script directory**
+
+The script directory contains bash scripts that can be used to install and configure Camunda 8.
+
+```
+cd camunda-deployment-references/aws/ec2/scripts
+```
+
+2. **Advanced configuration options**
+
+There are certain script features hidden behind a feature flag. Those are the following:
+
+- `CLOUDWATCH_ENABLED`: The default is false. If set to true will install the CloudWatch agent on each EC2 instance and export Camunda logs and Prometheus metrics to AWS CloudWatch.
+- `SECURITY`: The default is false. If set to true will use self-signed certificates to secure cluster communication, based on the procedure described in the [documentation](/self-managed/zeebe-deployment/security/secure-cluster-communication.md). This requires a manual step as a prerequisite as described below in step 3.
+
+Additionally, certain variables can be configured in the `camunda-install.sh` script to overwrite the default for Camunda and Java versions:
+
+- `OPENJDK_VERSION`: The Temurin Java version.
+- `CAMUNDA_VERSION`: The Camunda 8 version.
+- `CAMUNDA_CONNECTORS_VERSION`: The Camunda 8 connectors version.
+
+Setting those as environment variables will not take effect since the scripts are executed on the remote host with no access to your local environment.
+
+3. **(optional) Security**
+
+If you decide to enable `SECURITY`, please execute the `generate-self-signed-cert-authority.sh` script to once create a certificate authority. It would be wise to save those somewhere securely as you'll require those if you want to upgrade or change configs in an automated way. Worst case, you'll have to recreate the certificate authority certs via the script and all manually created client certificates.
+
+:::note
+Self-signed certificates are advocated for development and testing purposes. Check the [documentation](/self-managed/zeebe-deployment/security/secure-cluster-communication.md) on secure cluster communication to learn more about PEM certificates.
```

Comment:
It would require people to rotate the certificate every ~90 days. In a manual context maybe not ideal, especially for client/gateway certificates.
If we were properly testing this, would include it but until then would leave it out for now.

Reply:
this is about encryption of broker to broker (gateway) or gateway to client communication. A feature on the Zeebe side.
I'm no sure we detail it at all anywhere on how to do that.